triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractTriggerTestWithCanton.scala

```diff
@@ -9,7 +9,7 @@ package test
 import java.util.UUID
 import akka.stream.scaladsl.{Sink, Source}
 import com.daml.bazeltools.BazelRunfiles
-import com.daml.ledger.api.refinements.ApiTypes
+import com.daml.ledger.api.refinements.ApiTypes.Party
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument